.travis.yml

```diff
@@ -58,9 +64,6 @@ matrix:
         local-dir: "/tmp/ps-release"
         on:
           all_branches: true
-  allow_failures:
-    - php: 7.2
-      env: EXTRA_DEPS=phpHigh  PRESTASHOP_TEST_TYPE=unit
```

Comment:
Why are you removing this tests?

Reply:
Because it forces to update all our dependencies without considering the platform, the php compatibility or the dependency compatibility. For example, it currently update `Pelago\Emogrifier\HtmlProcessor` and break everything, and if you update the code for this dependency, another one's broken again, etc. https://travis-ci.com/PrestaShop/PrestaShop/jobs/254513713#L651

It should have been useful if the update was only for Twig2, and all our dependencies.
Since we never check this one, it's consider as a useless travis job

Reply:
okay then